en/firefox/whatsnew/whatsnew-fx80.ftl

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/80.0/whatsnew/all/
+
+# html page title
+whatsnew80-page-title = What’s new with { -brand-name-firefox }
+
+whatsnew80-up-to-date-banner = Congrats! You’re using the latest version of { -brand-name-firefox }.
+
+# Main title
+whatsnew80-main-title = Do just about everything, a little faster
+whatsnew80-main-intro = Move from computer to phone without missing a beat. Get { -brand-name-firefox } for mobile.
+whatsnew80-primary-cta-description = Enter your email or phone number for a download link
+
+# content buckets below hero
+whatsnew80-password-fatigue = Password fatigue is real
+whatsnew80-the-average-person = The average person has 130 online accounts. { -brand-name-firefox } for mobile remembers and stores your passwords safely across synced devices, so you can log in with just one click.
+
+whatsnew80-privacy-you-never = Privacy you never have to ponder
+whatsnew80-firefox-for-mobile = { -brand-name-firefox } for mobile blocks most online ad trackers automatically so there’s no need to dig into your security settings.
+
+whatsnew80-new-firefox-android = New { -brand-name-firefox } for Android
+whatsnew80-our-latest-version = Our latest version of { -brand-name-firefox } features lighting-fast page loads and a clean new design that makes it easier to get more things done, more quickly.
```

Comment:
`lighting-fast` should be `lightening-fast`.

Reply:
Good catch!